charts/gardener-dashboard/values.yaml

```diff
@@ -157,13 +157,20 @@ global:
     #   # - `${projectName}`: name of the project
     #   # - `${utcDateTimeNow}`: current date-time in utc format
     #   # - `${seedName}`: shoot's seed name
+    #   # - `${accessRestrictions}`: shoot's selected accessRestrictions
     #   issueDescriptionTemplate: |
     #     ## Which cluster is affected?
 
     #     `Cluster Details Dashboard Link`: [${projectName}/${shootName}](${shootUrl})
     #     `Operating System`: ${machineImageNames}
     #     `Platform`: ${providerType}
-
+    #     <% if(accessRestrictions.length) { %>
```

Comment:
Is this a good place for documentation of such complicated things. I would propose to create a standalone docs page for this topic and refer it by link from here.